File: Baekjoon/Greedy/2720.py
# https://www.acmicpc.net/problem/2720

import sys
input = sys.stdin.readline

t = int(input())
arr = []

def change(n):
    result = []
    coin = [25, 10, 5, 1]
    
    for i in coin:
        result.append(n // i)
        n = n % i
    
    return ' '.join(map(str, result))
    
for _ in range(t):
    n = int(input())
    arr.append(change(n))
    
for i in arr:
    print(i)

File: Baekjoon/Greedy/2720.cpp
// https://www.acmicpc.net/problem/2720

#include <iostream>
using namespace std;

int main() {
    int T;
    cin >> T;
    
    while (T--) {
        int C;
        cin >> C;
        int Q = C / 25;
        C %= 25;
        int D = C / 10;
        C %= 10;
        int N = C / 5;
        C %= 5;
        int P = C;
        cout << Q << ' ' << D << ' ' << N << ' ' << P << '\n';
    }

    return 0;
}